gpc-api-facade/src/integrationTest/java/uk/nhs/adaptors/pss/gpc/controller/PatientTransferControllerIT.java

```diff
@@ -157,6 +157,19 @@ public void sendPatientTransferRequestWithInvalidBody() throws Exception {
             .andExpect(content().json(expectedResponseBody));
     }
 
+    @Test
+    public void sendPatientTransferRequestWithEmptyBody() throws Exception {
+        var expectedResponseBody = readResourceAsString(
+            "/responses/migrate-patient-record/unprocessableEntityEmptyBodyResponseBody.json");
+
+        mockMvc.perform(
+            post(MIGRATE_PATIENT_RECORD_ENDPOINT)
+                .contentType(APPLICATION_FHIR_JSON_VALUE)
+                .content(""))
```

Comment:
Maybe would be better if "" was StringUtils.EMPTY instead? 
```suggestion
                .content(StringUtils.EMPTY))
```